File: 백준/Silver/5671. 호텔 방 번호/호텔 방 번호.py
import sys
input = sys.stdin.readline

while True:
    a = input().strip()
    if not a:
        break
    n, m = map(int, a.split())

    cnt = 0
    for i in range(n, m+1):
        if len(str(i)) == len(set(str(i))):
            cnt += 1

    print(cnt)

File: 백준/Silver/5671. 호텔 방 번호/호텔 방 번호.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "./example.txt")
  .toString()
  .trim()
  .split("\n")
  .map((v) => v.split(" ").map(Number));

const [...s] = input;
for (let v of s) {
  const [m, n] = v;

  let res = 0;
  for (let i = m; i <= n; i++) {
    if (String(i).length === [...new Set(String(i))].length) res += 1;
  }

  console.log(res);
}
